### PR TITLE
Formatting in Jupyter Notebooks

### PR DESCRIPTION
**Pull request by amandarichardsonn**
Jupyter notebook math expressions were not rendering locally or in docker container - to test run make docs and make docks.

1. Added the `sphinx.ext.mathjax` extension in conf.py
2. Set mathjax_path: path to the MathJax.js file

### CODE REVIEW

**Review by mellis13 — DISMISSED** [review]

File: doc/conf.py [hunk 1]
```diff
@@ -62,6 +63,9 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+# The path to the MathJax.js file that Sphinx will use to render math expressions
+mathjax_path = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js'
```
[the remark this answers is not in the document]
Comment (amandarichardsonn) [reply doc/conf.py:67]: Yes, you can find it [here](https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js)

[the remark this answers is not in the document]
Comment (mellis13) [reply doc/conf.py:67]: Thank you!

[the remark this answers is not in the document]
Comment (amandarichardsonn) [reply doc/conf.py:67]: I updated the link a couple seconds ago btw!

**Review by al-rigazzi — APPROVED** [review]
LGTM, the docs are getting better and better!

**Pull request merged**